test/test.js

```diff
@@ -124,6 +124,7 @@ mapSeries(directories, readdir, function (err, directoryFiles) {
       if (directory === 'test' && file === 'test.js') return
       if (extname(file) !== '.js') return
       if (file[0] === '_') return
+      if (file.indexOf('.test.') === -1) return
```

Comment:
nit:

```suggestion
      if (!file.endsWith('.test.js')) return
```

Reply:
ha -- I auto committed this and the lint failed because the `var extname = path.extname` line was no longer needed.  I fixed that up locally -- @trentm restamp when the tests are done?